manifest.json

```diff
@@ -8,16 +8,17 @@
         "es": "DokuWiki es un sistema de Wiki de uso sencillicimo y compatible con los estándares.",
         "it": "DokuWiki è un Wiki aderente agli standard, semplice da usare, finalizzato principalmente alla creazione di documentazione di qualsiasi tipo."
     },
-    "developer": {
+    "url": "http://zeropi.net",
```

Comment:
oups... i updated lots of apps at the same time, i went to quick on this one.
url shall be a link to the upstream project website if it exists.